src/lang/english.txt

```diff
@@ -2859,7 +2859,7 @@ STR_HOUSE_PICKER_YEARS_FROM                                     :{BLACK}Years: {
 STR_HOUSE_PICKER_YEARS_UNTIL                                    :{BLACK}Years: {ORANGE}Until {NUM}
 STR_HOUSE_PICKER_SIZE                                           :{BLACK}Size: {ORANGE}{NUM}x{NUM} tiles
 STR_HOUSE_PICKER_CARGO_ACCEPTED                                 :{BLACK}Cargo accepted: {ORANGE}
```

Comment:
Why shouldn't accepted cargo use the same form?

Reply:
Accepted cargo supports fractional amounts and so is built differently. CARGO_LIST only supports yes/no.